pandas/io/formats/style.py

```diff
@@ -2831,3 +2785,166 @@ def _highlight_between(
         else np.full(data.shape, True, dtype=bool)
     )
     return np.where(g_left & l_right, props, "")
+
+
+def _bar(
```

Comment:
could consider moving style* things to a separate dir e.g. pandas/io/formats/style/* and then have flexibility to for example move some utilities to own modules. (obviously for future)